examples/apps/heart_disease_eval/README.md

```diff
@@ -21,6 +21,8 @@ The command line client, `Demo.py`, runs a set of pre-built JSON requests.
 To run, follow the instructions in the "Testing" section of the
 [build document](../../../BUILD.md#testing).
 You may run the CLI demo in standalone mode or in a Docker container.
+Heart disease workload is designed to compute average heart disease risk, 
+even when invalid data is passed at in_data parametere via command prompt.
```

Comment:
Change 
`passed at in_data parametere via` 
to
`passed in the in_data parameter via the`

Reply:
done